cmd/server/fiber_server.go

```diff
@@ -119,19 +119,19 @@ func (s *FiberHttpServer) initUserRouter(router fiber.Router, httpHandler handle
 	userRouter := router.Group("/users")
 
 	userRouter.Get("/", httpHandler.User().GetAllUsers)
-	userRouter.Post("/", httpHandler.Middleware().IsLogin, httpHandler.Middleware().SuperAdmin, httpHandler.User().CreateUser)
-	userRouter.Patch("/", httpHandler.Middleware().IsLogin, httpHandler.User().UpdateProfile)
-	userRouter.Put("/:user_id", httpHandler.Middleware().IsLogin, httpHandler.Middleware().SuperAdmin, httpHandler.User().UpdateUserByID)
+	userRouter.Post("/", httpHandler.User().CreateUser)
 }
 
 func (s *FiberHttpServer) initAttachmentRouter(router fiber.Router, httpHandler handlers.Handler) {
 	attachmentRouter := router.Group("/attachments")
 
-	attachmentRouter.Post("/:document_id", httpHandler.Attachment().CreateAttachments)
+	attachmentRouter.Get("/:document_id", httpHandler.Attachment().CreateAttachments)
 }
 
 func (s *FiberHttpServer) initDocumentRouter(router fiber.Router, httpHandler handlers.Handler) {
-	documentRouter := router.Group("/documents")
+	documentsRouter := router.Group("/documents")
+
+	documentsRouter.Post("/", httpHandler.Document().CreateDocument)
+	documentsRouter.Delete("/:document_id", httpHandler.Document().DeleteDocumentByID)
```

Comment:
For post/delete routes, let's add some middlewares to e.g.
```go
documentsRouter.Post("/", httpHandler.Middleware().IsLogin, httpHandler.Middleware().SuperAdmin, httpHandler.Document().CreateDocument)
documentsRouter.Delete("/:document_id", httpHandler.Middleware().IsLogin, httpHandler.Middleware().SuperAdmin, httpHandler.Document().DeleteDocumentByID)
```
If you have any problems trying to test routes that requires login, we can talk na